utils/plot_utils.py

```diff
@@ -194,14 +194,18 @@ def plot_HDR_curves(predicted_hdr, empirical_hdr, output_image_name, title, ax=N
     plt.savefig(output_image_name)
     plt.show()
 
-def plot_calibration_curves(conf_levels, unc_pcts, cal_pcts, output_image_name):
+def plot_calibration_curves(conf_levels, unc_pcts, cal_pcts, output_image_name, cal_conformal=False):
     """
     Plot calibration curves
     """
     plt.figure(figsize=(10,7))
     plt.plot([0,1],[0,1],'--', color='grey')
-    plt.plot(1-conf_levels, unc_pcts, '-o', color='purple', label='Uncalibrated')
-    plt.plot(1-conf_levels, cal_pcts, '-o', color='red', label='Calibrated')
+    if cal_conformal:
```

Comment:
Oh, looks weird to have conf or 1-conf; i think we should stick with just one (independently of the method). Let's keep it this way for the moment.